.github/workflows/sphinx-build.yml

```diff
@@ -10,6 +10,20 @@ on:
     - main
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: adopt
+      - name: Run Digital Unit Tests
+        run: |
+          cd schematics
+          java -cp Digital.jar CLI test *.dig
```

Comment:
imo simply `java -cp Digital.jar CLI test schematics/*.dig` -- doing `cd` first groups the actions output under a `cd` tag which looks wonky.

Reply:
Or rather, `run: java -cp Digital.jar CLI test **/*.dig` -- there's no point in keeping all schematics in `schematics`, they will likely be tied to topics/assignments, so should likely be grouped with them, similar to images.

Reply:
As of now, I specified `site/topics/**/*.dig`. 

The reasoning is
1. I do not know how I will ultimately layout the schematics
2. Some schematics will have unit tests that fail since they would me empty files with working unit tests. The idea would be that students fill in the schematic and run the provided tests (though, there is a way to specify a separate testing file, which may make this point moot). 
3. The path makes the action work.